src/k8s-engine/wsl.ts

```diff
@@ -13,6 +13,7 @@ import util from 'util';
 import semver from 'semver';
 
 import INSTALL_K3S_SCRIPT from '@/assets/scripts/install-k3s';
+import INSTALL_WSL_HELPERS_SCRIPT from '@/assets/scripts/install-wsl-helpers';
```

Comment:
The `@` is a [webpack alias](https://github.com/rancher-sandbox/rancher-desktop/blob/ac9544f071fc1a856eb79b193a757433e7c0d8a6/scripts/lib/build-utils.mjs#L145).

This import works because we use [raw-loader](https://github.com/rancher-sandbox/rancher-desktop/blob/ac9544f071fc1a856eb79b193a757433e7c0d8a6/scripts/lib/build-utils.mjs#L176) for things in `assets/scripts/`.